docs/topics/rn-new-features.adoc

```diff
@@ -7,21 +7,46 @@
 
 This section describes the new features of the {ProductName} ({ProductShortName}) {ProductVersion}.
 
-.MTA plugin for IntelliJ IDEA
+.Rules for migrating from Java EE8 to Jakarta EE9
+{ProductShortName} now includes a comprehensive set of rules to support the Java EE8 to Jakarta EE9 migration path by:
```

Comment:
Lead-in should be a complete sentence.

Reply:
@apinnick Suggestion: "{ProductShortName} now includes a comprehensive set of rules to support the Java EE8 to Jakarta EE9 migration path. These rules:
* Replace ...

Reply:
Also, put a blank line between header and text.

Reply:
In fact, I would combine the "replacing" bullets into one bullet and the "renaming" bullets into one bullet

Reply:
Maybe this:

{MTA} rules replace Java EE8 artifact dependencies and package import statements and rename Java EE8 XML schema namespaces, prefixed properties, and bootstrapping files for Jakarta EE9.